### DISCUSSION TITLE
Why is this plugin renaming all subplugins to ff2?

### DISCUSSION THREAD

**Discussion by bottiger1** [Batfoxkid/Freak-Fortress-2-Rewrite#74] [Q&A]
Can this behavior be disabled?

**Comment by Batfoxkid** [comment]
The plugin uses .ff2 to prevent SourceMod loading plugins naturally and preserved as a memorable name for FF2 subplugins.

**Comment by bottiger1** [comment]
Can this behavior be disabled since we use the disabled folder?

**Reply by Batfoxkid** [reply]
This could be the case as an auto detection if the disabled folder is used as the subplugin folder.
